tox.ini

```diff
@@ -4,8 +4,11 @@ skip_missing_interpreters = True
 
 [testenv]
 install_command = pip install --process-dependency-links {opts} {packages}
-deps = .[docs,test]
 commands =
+# install documentation and testing requirements
+# NOTE: Don't use 'deps = .[docs,test]' tox option since we want Tox to install
+# the package from sdist first
+    pip install --process-dependency-links .[docs,test]
```

Comment:
I fully agree that the `master` branch should only depend on officially released packages. What I was thinking was more a convenience for developers that would sometimes need to bump a certain dependency to the latest git version while developing on a personal feature branch.

Anyway, I think we can omit `--process-dependency-links` here and I'll update the PR.